junction/templates/pages/home_page.html

```diff
@@ -36,9 +36,14 @@ <h2 class="heading">
         {% if not conferences %}
             <p> No conferences found. </p>
         {% endif %}
+    {% for year, c in conferences.items %}
+            <h3>
+                    PyCon India {{ year }}
```

Comment:
IMHO, we should not assume Junction is always going to be used for PyCon India. :)